Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -292,7 +293,7 @@ verifyRC isDashboard mbMerchant (personId, _, merchantOpCityId) req bulkUpload =
           unless (extractRCNumber == rcNumber) $
             throwImageError req.imageId $ ImageDocumentNumberMismatch (maybe "null" maskText extractRCNumber) (maybe "null" maskText rcNumber)
         Nothing -> throwImageError req.imageId ImageExtractionFailed
-
+  whenJust mbFleetOwnerId $ Redis.set (makeFleetOwnerKey req.vehicleRegistrationCertNumber)
```

Comment:
Have doubts about this approach, because image extraction will happen under certain conditions

```
  when
    ( isNothing req.vehicleDetails && isNothing req.dateOfRegistration && documentVerificationConfig.checkExtraction
        && (not isDashboard || transporterConfig.checkImageExtractionForDashboard)
        && (not bulkUpload)
    )
```

e.g. its will not work for bulk upload